mmv1/templates/terraform/examples/dns_managed_zone_private_multiproject.tf.erb

```diff
@@ -0,0 +1,63 @@
+resource "google_dns_managed_zone" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['zone_name'] %>"
+  dns_name    = "private.example.com."
+  description = "Example private DNS zone"
+  labels = {
+    foo = "bar"
+  }
+
+  visibility = "private"
+
+  private_visibility_config {
+    networks {
+      network_url = google_compute_network.network-1_project-1.id
+    }
+    networks {
+      network_url = google_compute_network.network-2_project-1.id
+    }
+    networks {
+      network_url = google_compute_network.network-1_project-2.id
+    }
+    networks {
+      network_url = google_compute_network.network-2_project-2.id
+    }
+  }
+}
+
+resource "google_project" "project-1" {
```

Comment:
Error: 

```
Error: Reference to undeclared resource
        
          on terraform_plugin_test.tf line 38, in resource "google_compute_network" "network-1_project-1":
          38:   project                 = google_project.project_1.project_id
        
        A managed resource "google_project" "project_1" has not been declared in the
        root module.
        
```

All of these need to use underscores to match the references to them.

```suggestion
resource "google_project" "project_1" {
```

Reply:
🤦‍♂️ Thanks @melinath. Very nice catch!